modules/graphql_content/src/Plugin/Deriver/RawValueFieldItemDeriver.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\graphql_content\Plugin\Deriver;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\graphql_content\Plugin\GraphQL\Types\RawValueFieldType;
+
+class RawValueFieldItemDeriver extends FieldDeriverBase {
+
+  // TODO Provide more mappings.
```

Comment:
I think this should be moved to a place where we can make it more extendable. E.g. a service parameter?